docs/user-guide/src/capm3/installation_guide.md

```diff
@@ -11,7 +11,7 @@ install them yourself.
 
 1. Install `clusterctl`, refer to Cluster API [book](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) for installation instructions.
 1. Install `kustomize`, refer to official instructions [here](https://kubectl.docs.kubernetes.io/installation/kustomize/).
-1. Install Ironic, refer to [TODO](TODO).
+1. Install Ironic, refer to [this page](https://book.metal3.io/ironic/ironic_installation.html).
```

Comment:
I think it would be better to do a relative link here instead. Then it will be correct also if the domain changes (e.g. in previews generated for each PR).
```suggestion
1. Install Ironic, refer to [this page](../ironic/ironic_installation.html).
```